packages/wp-dashboard/src/components/editorSettings/adManagement/index.js

```diff
@@ -199,6 +212,7 @@ AdManagement.propTypes = {
   publisherId: PropTypes.string,
   adSenseSlotId: PropTypes.string,
   adManagerSlotId: PropTypes.string,
+  mgidWidgetId: PropTypes.string,
```

Comment:
If `mgidWidgetId` is only going to accept numbers, may we change it from `string` to `number`?

```suggestion
  mgidWidgetId: PropTypes.number,
```